src/app/zap-templates/zcl/data-model/chip/channel-cluster.xml

```diff
@@ -74,7 +74,7 @@ limitations under the License.
     <item name="MSO" value="0x00"/>
   </enum>
 
-  <enum name="ChannelStatusEnum" type="enum8">
+  <enum name="StatusEnum" type="enum8">
```

Comment:
Worth adding a compat `using` in the backwards compat enums header?

Reply:
I was not aware of that header, am now and will add it.